framework/src/modules/dpos_v2/events/delegate_banned.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BaseEvent, EventQueuer } from '../../base_event';
+import { ADDRESS_LENGTH } from '../constants';
+
+export interface DelegateBannedEventData {
+	address: Buffer;
+	height: bigint;
+}
+
+export const delegateBannedDataSchema = {
+	$id: '/pos/events/delegateBannedData',
+	type: 'object',
+	required: ['address', 'height'],
+	properties: {
+		address: {
+			dataType: 'bytes',
+			fieldNumber: 1,
+			minLength: ADDRESS_LENGTH,
+			maxLength: ADDRESS_LENGTH,
```

Comment:
for address, please use `format: 'lisk32'`

Reply:
for the lisk32 format, the length is checked for the format, so no need to add `min/maxLength`